src/telemetry/events.ts

```diff
@@ -26,3 +27,8 @@ export function logUsage(event: UserEvent, data?: Record<string, any | Telemetry
   // May or may send to Segment based on user settings. See checkTelemetrySettings().
   getTelemetryLogger().logUsage(event, data);
 }
+
+/** Compute a hex-encoded SHA-256 hash of a client-side string to accompany telemetry data. */
+export function hashed(input: string): string {
```

Comment:
Will be used by other PRs coming up, such as for hashing a schema body for schema-related telemetry events.